Assets/Scripts/StandardSamples/UI/Storage/Player/UIPlayerDataStorageMenu.cs

```diff
@@ -302,6 +302,13 @@ public override void Hide()
 
             currentSelectedFile = string.Empty;
             currentInventory = null;
+            PlayerDataStorageService.Instance.OnFileListUpdated -= UpdateFileListUI;
```

Comment:
I am fairly certain that we want to stay subscribed to the event during the file list update - because when a sample is "hidden," we still want it to get updated when appropriate so that when it is subsequently "shown," it's UI is up-to-date.

Reply:
Per conversation I am going to resolve this conversation - the need to unsubscribe is a peculiarity of the Unity game object lifecycle system.